server/src/main/java/org/apache/iotdb/db/engine/compaction/CompactionTaskManager.java

```diff
@@ -155,21 +138,23 @@ public void waitAndStop(long milliseconds) {
   public void waitAllCompactionFinish() {
```

Comment:
what's the behavior of this function? When others call this, will more compaction tasks be submitted? There are three candidateCompactionTaskQueue.clear(), which is unnormal.

Reply:
resolve